lambda_backend/route_microservice/src/common/schema.ts

```diff
@@ -5,48 +5,46 @@ const CapitalizedAlphabets = '^[A-Z]*$';
 const NumericDecimalCommaSpace = '^[0-9., ]*$';
 const AsciiCharacters = '^[ -~]+$';
 
+const countryCode = { type: 'string', maxLength: 3, pattern: CapitalizedAlphabets };
```

Comment:
Maybe should change the regex to `+` instead of `*` for `AlphanumericSpace`, `CapitalizedAlphabets` and `NumericDecimalCommaSpace`